src/components/ContentBlock.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react';
+import { PseudoBox, Box, Flex, theme } from '@chakra-ui/core';
+
+const fullWidthOverlayOpacity = 0.3;
+
+const formatHexCode = hexCode => {
```

Comment:
Curious, what was the scenario that this utility is for?

Reply:
There are different ways you can pass in the color. 

Using a theme variable, it'll pull the hex from the theme config, and then convert it into the 6 digit hex if it isn't already.

Using a 3 digit hex, it'll convert it into the 6 digit hex.

This is so I can have a consistent format to apply the 2 extra digits for the alpha in the linear-gradient for the fade scenarios.